### DISCUSSION TITLE
Some suggestions regarding mod presets

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19692] [Ideas]
- Ability to right click and rename mod presets and their descriptions

This isn't something that would be strictly required since it is so easy to create mod presets, but I think that it would be a nice quality of life feature that probably wouldn't be too difficult to add.

- Ability to sort mod presets manually (similar to how you can sort pinned scores on the website)

Currently, mod presets are sorted alphabetically which is fine for the most part, but I think that it would be nice to be able to sort them manually. If you want your mod presets to be in a certain order, you usually have to add a number at the beginning of each of the names of the mod presets and while it works, it isn't ideal.

**Comment by nagi-desuuu** [comment]
- This might require a UX pass first, but it could be implemented once multi-player gets the presets (I think)
- That's actually not a bad idea... Maybe I'll look forward into this since code-wise it may be easy to implement.

**Reply by nagi-desuuu** [reply]
Actually, I take the framework pull back...
I'll move it here just in case: https://github.com/ppy/osu-framework/pull/5255
